src/pages/Postcreatepage/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import  { Redirect } from 'react-router-dom';
+import { Form } from 'react-bootstrap';
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import { createPost } from 'store/actions/createPost';
+import { Styled } from 'pages/Postcreatepage/Postcreatepage.styles';
+import CommonLayout from 'layout/CommonLayout';
+import CustomInputComponent from 'components/Input';
+import CustomButton from 'components/Button';
+
+
+const Postcreatepage = ( props: any ) => {
```

Comment:
Use like this CamelCase ( PostCreatePage )

Reply:
Used Camel Case.